src/Analyzers/Core/Analyzers/SimplifyInterpolation/Helpers.cs

```diff
@@ -142,12 +142,12 @@ private static TextSpan GetSpanWithinLiteralQuotes(IVirtualCharService virtualCh
                             {
                                 var alignmentSyntax = alignmentOp.Syntax;
 
-                                unwrapped = invocation.Instance;
+                                unwrapped = invocation.Instance!;
```

Comment:
Should be an assertion unless there's a relevant test elsewhere in the method.
For the one below, `invocation.Instance!.Syntax.FullSpan`, the suppression also deviates from our current guideline. @cston was proposing that we allow it, but if I recall correctly we need to further discuss with @jarepar.

Reply:
There is. See `HasImplicitInstance` a few lines above.

Reply:
Got it. Thanks